### ISSUE TITLE
Page isn't redirecting properly

### ISSUE THREAD

**Issue by ndarilek** [jparyani/roundcubemail#9]
Seeing Firefox's standard "The page isn't redirecting properly" error message on multiple grains, both old and new, on two different Sandstorm servers. Nothing is in the grain logs. Unfortunately, none of my Roundcube grains work anymore.

**Comment by jparyani** [comment]
What version of Firefox/OS are you using?

**Comment by ndarilek** [comment]
Fedora 24, Firefox 50.0.

**Comment by jparyani** [comment]
I'm unable to reproduce this on Firefox v50.1.0 on OSX.

I was just trying a brand new grain. Did you do anything special to trigger this? Can you try it in an incognito window to see if it reproduces there?

**Comment by ndarilek** [comment]
Yeah, looks like a private tab fixes this, along with another related 
issue I had with Sandpass. Thanks for the recommendation.

Interestingly enough, the behaviors for which switching to a private tab 
helped are on three different Sandstorm servers, one of which was a new 
install as of a couple weeks ago, and another of which was Oasis. I 
guess I can figure out how to clear all of Firefox's information and 
storage for these specific sites, but I'm wondering if there's anything 
I might provide to debug the root cause before I wipe everything?

**Comment by jparyani** [comment]
My guess is that it's cookie related. We've had some people report very similar issues happening due to cookies with TinyTinyRSS. If you could look at what cookies are set on sandstorm.io/oasis.sandstorm.io, that would be very helpful. Sandstorm itself doesn't use cookies so they should all be only set by apps. 

You can email support at sandstorm.io if you're not comfortable sharing it publicly.

**Comment by ndarilek** [comment]
OK, I've confirmed that deleting my cookies seems to resolve this by 
deleting them on one of the failing instances. Now everything works 
again. :)

Oasis is still in failure mode, though. Any recommendations on how to 
export cookies from Firefox? I have an Export Cookies addon, but it 
seems to dump *all* of my cookies to a Netscape-style cookies.txt. I can 
edit them down, but the long alphanumeric hostnames are hell to listen 
to with my screen reader in order to find the applicable ones, and there 
doesn't appear to be a way to export cookies from Firefox's cookie list 
in Preferences.

**Comment by jparyani** [comment]
Hmm sorry, I don't actually know the best way to do this. I typically use Chrome, and just manually look them up through the dev tools inspector.
